happypanda/gallerydb.py

```diff
@@ -27,15 +27,15 @@
 
 from PyQt5.QtCore import QObject, pyqtSignal, QTime
 
-from utils import (today, ArchiveFile, generate_img_hash, delete_path,
+from happypanda.utils import (today, ArchiveFile, generate_img_hash, delete_path,
```

Comment:
actually almost all of your change pass pep8 check. so it is kinda left out(?). ignoring this will add things to pep8 fix for on future commits